waspc/src/Wasp/Util.hs

```diff
@@ -78,11 +78,9 @@ camelToKebabCase camel@(camelHead : camelTail) = kebabHead : kebabTail
     isCamelHump (a, b) = (not . isUpper) a && isUpper b
 
 kebabToCamelCase :: String -> String
-kebabToCamelCase = concat . capitalizeAllWordsExceptForTheFirstOne . wordsBy (== '-')
-  where
-    capitalizeAllWordsExceptForTheFirstOne :: [String] -> [String]
-    capitalizeAllWordsExceptForTheFirstOne [] = []
-    capitalizeAllWordsExceptForTheFirstOne (firstWord : otherWords) = firstWord : map toUpperFirst otherWords
+kebabToCamelCase kebabCaseStr = case wordsBy (== '-') kebabCaseStr of
+  [] -> ""
+  (firstWord : otherWords) -> concat $ firstWord : map toUpperFirst otherWords
```

Comment:
Has nothing to do with the PR, but I couldn't resist.

Reply:
That's true. I regressed (too much Haskell I guess). I changed it back to the original with a twist.